dali/operators/decoder/nvjpeg/nvjpeg_decoder_decoupled_api.h

```diff
@@ -487,10 +505,10 @@ class nvJPEGDecoder : public Operator<MixedBackend>, CachedDecoderImpl {
       const auto in_size = in.size();
 
       SampleData &data = sample_data_[i];
-      data.file_name = in.GetSourceInfo();
+      data.clear();
```

Comment:
It is counterintuitive that `clear` doesn't affect `sample_idx`.

Reply:
that's true

Reply:
fixed